README.md

```diff
@@ -236,6 +237,35 @@ end
 
 You can access user model through authenticatable.
 
+### Claiming tokens
+
+Opt-in for marking tokens as `claimed` so that they can't be used again.
+
+config/initializers/passwordless.rb
+
+```ruby
+# Default is `false`
+Passwordless.claim_token_after_sign_in = true
```

Comment:
I think something like `restrict_token_reuse` is more descriptive. What do you think?

Reply:
Sounds good!